chart/nginx-templates/default.conf.template

```diff
@@ -24,16 +24,24 @@ server {
   listen ${PORT};
   listen [::]:${PORT};
   server_name ${HOST};
+  set $assets_directory ${ASSETS_DIRECTORY};
+  set $cached_assets_directory ${CACHED_ASSETS_DIRECTORY};
 
   location /openapi.json {
     return 307 https://raw.githubusercontent.com/huggingface/datasets-server/main/${OPENAPI_FILE};
   }
 
   location /assets/ {
+    if ($assets_directory = false) {
```

Comment:
Not sure if this is the best way to do it:
- If ASSETS_DIRECTORY is set, it should take the alias with the value but if not, it should not consider the alias. We need this logic because in prod, we don't need this redirect, but for e2e and local dev env, we use local storage.
Same for cached-assets.